crates/bevy_winit/src/winit_windows.rs

```diff
@@ -20,18 +21,24 @@ use crate::{
     converters::convert_window_level,
 };
 
+/// A resource which maps window entities to [`winit`] library windows.
 #[derive(Debug, Default)]
 pub struct WinitWindows {
```

Comment:
Sorry to come bakc after this PR is merged; but is this documentation accurate ? as the struct doesn't derive `Resource`, it's not a resource 🤔 ?

Reply:
oh it's a `NonSendMut` :+1: